src/tools/rust-analyzer/crates/hir-expand/src/inert_attr_macro.rs

```diff
@@ -486,7 +486,7 @@ pub const INERT_ATTRIBUTES: &[BuiltinAttribute] = &[
         rustc_legacy_const_generics, Normal, template!(List: "N"), ErrorFollowing,
         INTERNAL_UNSTABLE
     ),
-    // Do not const-check this function's body. It will always get replaced during CTFE.
+    // Do not const-check this function's body. It will always get replaced during CTFE via `hook_special_const_fn`.
```

Comment:
There's no such function in RA so does this comment really make sense here?

Reply:
Yes, the file is largely a copy of the rustc one.